contracts/src/oracle/Oracle.cairo

```diff
@@ -220,3 +240,21 @@ func get_admin_address{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_c
     let (admin_address) = Proxy.get_admin_address()
     return (admin_address)
 end
+
+@external
+func set_checkpoint{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    key : felt, aggregation_mode : felt
+):
+    Oracle.set_checkpoint(key, aggregation_mode)
+    return ()
+end
+
+@external
+func set_sources_threshold{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    threshold : felt
+):
+    Oracle.only_oracle_controller()
```

Comment:
should be `assert_only_admin`, oracle_controller is from when we used to have a separate oracle controller and implementation (before the proxy)